jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,16 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+mountpath="$WORKSPACE/RackHD/docker/files/mount"
+retrytimes=10
+while [ ! -d "$mountpath" ]
+do
+    echo "mount is not finished"
+    retrytimes=$(($retrytimes-1))
+    echo "check mount is successful , retry : $retrytimes times"
+    sleep 5
+done
+echo "change the authority of mount"
+echo $SUDO_PASSWORD |sudo -S chown -R $USER:$USER $WORKSPACE/RackHD
```

Comment:
please put comment in code to explain the motivation.
# change the mount folder owner from root to $USER. in case the script aborted and cleanup doesn't have chance to be invoked, and next run , Jenkins deleteDir will fail to delete workspace.